src/index.ts

```diff
@@ -622,7 +623,19 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
     eventSourceWithMiddleware = addMiddlewareToEventSource(middleware)
   }
 
-  return { on, off, variation, close, setEvaluations, registerAPIRequestMiddleware }
+  const refreshCache = (): boolean => {
+    // only fetch flags if enough time has elapsed to avoid pressuring backend servers
+    if (lastCacheRefreshTime == null || Math.round((Date.now()-lastCacheRefreshTime)/1000) >= 60) {
```

Comment:
We also don't want to do this if the SDK hasn't made it's initial fetch of evaluations.

Reply:
I don't understand what you mean here, you mean fetchFlags() shouldn't be called unless we're authenticated?

Reply:
That's not how this works. The customer will immediately have access to this new function. The call to auth is async and the invocation is not, so calling `initialize` will not wait to return.

```typescript
const x = initialize(/*...*/)
x.refreshEvaluations() // <-- this will get called before the auth call has completed
```

Reply:
ok thanks - i think we can silently return if we're not authenticated, is there a handy way to check we're authenticated? Looks like jwtToken gets set when auth completes. I guess we should check for closed too...

`if (!jwtToken || closed) return`

at the beginning

Reply:
We basically don't want to allow this until the `READY` would have been fired, specifically as a result of a fetch, not a cache load or `setEvaluations` call (those two will fire the `READY` event optimistically since they have evals, but the auth may or may not succeed).